src/i18n/strings/en_EN.json

```diff
@@ -120,17 +120,21 @@
   "zh-tw":"Chinese (Taiwan)",
   "zu":"Zulu",
   "A registered account is required for this action": "A registered account is required for this action",
+  "a room": "a room",
   "A text message has been sent to +%(msisdn)s. Please enter the verification code it contains": "A text message has been sent to +%(msisdn)s. Please enter the verification code it contains",
-  "accept": "accept",
```

Comment:
This originally formed part of the sentence "would you like to accept or decline" which we now use in the full JSX form. Independently from that, I needed to add in Accept/Decline buttons, hence the ultimate tense change here.